examples/demo13.py

```diff
@@ -0,0 +1,378 @@
+    # diameter and (since we didn't use any keyword arguments to modify this) using the typical
+    # exposure time for WFIRST images.
+    filters = wfirst.getBandpasses(AB_zeropoint=True)
+    # We care about the NIR imaging, not the prism and grism, so let's remove them from the dict
+    # containing the filters:
+    del filters['SNPrism']
+    del filters['BAO-Grism']
+    logger.debug('Read in WFIRST imaging filters.')
+
+    logger.info('Reading from a parametric COSMOS catalog.')
+    # Read in a galaxy catalog - just a random subsample of 500 galaxies for F814W<23.5 from COSMOS.
+    cat_file_name = 'real_galaxy_23.5_500_fits.fits'
+    dir = 'data'
+    # Use the routine that can take COSMOS real or parametric galaxy information, and tell it we
+    # want parametric galaxies.
+    cat = galsim.makeCOSMOSCatalog(cat_file_name, dir=dir, use_real=False)
+    logger.info('Read in %d galaxies from catalog'%len(cat))
+    # Just use a few galaxies, to save time.  Note that we are going to put 4000 galaxy images into
+    # our big image, so if we have n_use=10, each galaxy will appear 400 times.  Users who want a
+    # more interesting image with greater variation in the galaxy population can change `n_use` to
+    # something larger that evenly divides 4000 (but it should be <=500, which is the number of
+    # galaxies in the catalog).  The code is not set up to handle values of n_use that do not evenly
+    # divide n_tot, though it would not be very hard to do so.  With 4000 galaxies in a 4k x 4k
+    # image with the WFIRST pixel scale, the effective galaxy number density is 74/arcmin^2
+    n_use = 10
+    n_tot = 4000
+    n_per_gal = n_tot / n_use
+
+    # Here we carry out the initial steps that are necessary to get a fully chromatic PSF.  We use
+    # the getPSF() routine in the WFIRST module, which knows all about the telescope parameters
+    # (diameter, bandpasses, obscuration, etc.).  Note that we are going to arbitrarily choose a
+    # single SCA (Sensor Chip Assembly) rather than all of them, for faster calculations, and we're
+    # going to use a simple representation of the struts for faster calculations.  To do a more
+    # exact calculation of the chromaticity and pupil plane configuration, remove the
+    # `approximate_struts` and the `n_waves` keyword from the call to getPSF():
+    use_SCA = 7 # This could be any number from 1...18
+    logger.info('Doing expensive pre-computation of PSF.')
+    t1 = time.time()
+    PSFs = wfirst.getPSF(SCAs=use_SCA, approximate_struts=True, n_waves=10)
+    PSF = PSFs[use_SCA]
+    t2 = time.time()
+    logger.info('Done PSF precomputation in %.1f seconds!'%(t2-t1))
+
+    # Define the size of the postage stamp that we will use for each individual galaxy within the
+    # larger image, and for the PSF images.
+    stamp_size = 128
+
+    # We are going to choose a particular (RA, dec) location on the sky for our observation.
```

Comment:
Just a style point, but this would sound better in the imperative present tense: "Choose a particular...".  Simple present is also ok: "We choose a particular..."  Future tense makes it sound like you are going to do that later in the script, rather than on the very next line.